### DISCUSSION TITLE
Generated pom file of a submodule looses its parent with Tycho 4

### DISCUSSION THREAD

**Discussion by marcadetd** [eclipse-tycho/tycho#3024] [Q&A] [answered]
Hi,
I have a project with an Eclipse plugin as submodule. The pom file of the submodule has the pom file of the project as parent.
When I use Tycho 4 (4.0.0 or 4.0.4), the pom file installed with `mvn install` don't have anymore its parent element. 
```
<?xml version="1.0" encoding="UTF-8"?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>io.github.marcadetd</groupId>
  <artifactId>testdeploy</artifactId>
  <version>1.0.0-SNAPSHOT</version>
</project>
```
It is OK if I use Tycho 3.0.5:
```
<?xml version="1.0" encoding="UTF-8"?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <parent>
    <groupId>io.github.marcadetd</groupId>
    <artifactId>main</artifactId>
    <version>1.0.0-SNAPSHOT</version>
  </parent>
  <groupId>io.github.marcadetd</groupId>
  <artifactId>testdeploy</artifactId>
  <version>1.0.0-SNAPSHOT</version>
  <packaging>eclipse-plugin</packaging>
</project>
```
The installed pom file comes from tycho (`.tycho-consumer-pom.xml`).
Here is a small project demonstrating the problem: https://github.com/marcadetd/testmvn
Is there any new parameter that I've missed ? Should I open an issue ?
Thanks

**Comment by laeubi** [comment, marked as answer]
This is intentional because the parent of a project is usually not what one want to deploy, but you can configure this behavior here:

https://tycho.eclipseprojects.io/doc/latest/tycho-packaging-plugin/update-consumer-pom-mojo.html#removeParent

**Reply by marcadetd** [reply]
OK, thanks
